linux.md

```diff
@@ -7,29 +7,34 @@
    curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
    sudo apt-get update
    sudo apt-get install -y git nodejs python3
-   sudo npm install --global yarn
    ```
-   This uses apt to install Git, Node.js and Python and uses npm to install Yarn.<br><br>
-3. Copy each line in the following text, paste it in the terminal and hit return.<br>
+   This uses apt to install Git, Node.js and Python.<br><br>
+3. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
+    ```bash
+    sudo corepack enable
+    sudo corepack prepare pnpm@latest --activate
```

Comment:
do these need `sudo`?

Reply:
I talked with Kathi about the commands, and she tested them for me. She doesn't have to use `sudo`. 
Commit: [297d87c](https://github.com/upleveled/system-setup/pull/24/commits/297d87c25ac2b035a4a6c7107baa0da8faa4844c)